ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/Conversions.scala

```diff
@@ -215,4 +222,38 @@ private[state] object Conversions {
       )
     )
 
+  def encodeBlindingInfo(blindingInfo: BlindingInfo): DamlTransactionBlindingInfo =
+    DamlTransactionBlindingInfo.newBuilder
+      .addAllDisclosures(encodeDisclosure(blindingInfo.disclosure).asJava)
+      .addAllDivulgences(encodeDivulgence(blindingInfo.divulgence).asJava)
+      .build
+
+  private def encodeParties(parties: Iterable[Party]): List[String] =
+    parties.asInstanceOf[Set[String]].toList.sorted // Deterministic
```

Comment:
Can you remove these comments, please? I don't see why these help the reader. If you would like to stress out that the transaction blinding info is encoded in a deterministic way then please add that note to `encodeBlindingInfo`.

Reply:
Done, see #8024.